cognee/chunk_association.py

```
@@ -0,0 +1,115 @@
+    content: str
+    document_id: str
+    metadata: Dict[str, Any]
+
+@dataclass
+class Association:
+    source_chunk_id: str
+    target_chunk_id: str
+    weight: float
+    association_type: str = "semantic_similarity"
+
+class SimpleSimilarity:
```

Pylint (4.0.1)

[refactor] Too few public methods (1/2)

(R0903)